script.js

```diff
@@ -0,0 +1,118 @@
+//     Task 1
+
+let str = "true";
+let boo = false;
+let num = 17;
+let und = undefined;
+let nul = null;
+
+console.log(typeof str, typeof boo, typeof num, typeof und, typeof nul);
+
+//     Task 2
+
+let height = 15;
+let width = 20;
+
+if (height > width) {
+  console.log(height);
+} else {
+  console.log(width);
+}
+
+//     Task 3
+
+for (let i = 1; i <= 20; i++) {
+  if (i % 3 === 0) {
+    console.log(i);
+  }
+}
+
+//     Task 4
+
+let key = true;
+let documents = true;
+let pen = true;
+let apple = false;
+let orange = true;
+
+let shouldGoToWork = key && documents && pen && (apple || orange);
+console.log(shouldGoToWork);
+
+//     Task 5
+
+let numFB = prompt("Введите число");
+
+if (numFB % 5 === 0) {
+  console.log("Fiz");
+} else if (numFB % 3 === 0) {
+  console.log("Buz");
+} else if (numFB % 5 === 0 && numFB % 3 === 0) {
+  console.log("FizBuz");
+}
+
+//     Task 6
+
+let age = prompt("Укажите ваш возраст");
+
+if (age > 18) {
+  alert("Попей пивка");
+} else if (age >= 16 && age <= 18) {
+  akert("Можешь выкурить сигаретку, только маме не говори");
+} else {
+  alert("Пей колу");
+}
+
+//     Task 7
+
+let side = prompt(
+  "В какую сторону света вы хотели бы отправиться?"
+).toLocaleLowerCase();
+
+switch (side) {
+  case "юг":
```

Comment:
It will be better to move values of cases to the variable. And use them in the "switch" constuction.